serde/src/test/org/apache/hadoop/hive/serde2/columnar/TestLazyBinaryColumnarSerDe.java

```diff
@@ -165,7 +166,7 @@ public void testLazyBinaryColumnarSerDeWithEmptyBinary() throws SerDeException {
     outerStruct.mByte  = 101;
     outerStruct.mShort = 2002;
     outerStruct.mInt = 3003;
-    outerStruct.mLong = 4004l;
+    outerStruct.mLong = 40041;
```

Comment:
Probably, the original value is intended. Or we can use an uppercase.

```suggestion
    outerStruct.mLong = 4004L;
```